mzLib/Test/FileReadingTests/TestMsDataFile.cs

```diff
@@ -176,6 +227,17 @@ public void MoreMsDataFilesTests()
             Assert.IsNull(fakeDataFile.SourceFile.Id);
         }
 
+        [Test]
+        public void TestGetMsOrderByScanInDynamicConnection()
+        {
+            var file = new GenericMsDataFile(new MsDataScan[1],
+                new SourceFile("scan number only nativeID format",
+                    "mzML format",
+                    null, "SHA-1", @"C:\fake.mzML", null));
```

Comment:
please add comments explaining what you are testing.

Reply:
test is not included anymore